gammapy/stats/variability.py

```diff
@@ -289,9 +289,12 @@ def TimmerKonig_lightcurve_simulator(
     power_spectrum,
     npoints,
     spacing,
-    leakage_protection=1.0,
+    nchunks=10,
     random_state="random-seed",
     power_spectrum_params=None,
+    mean=0.0,
```

Comment:
Does this follow the initial Timmer and König paper? 
Are there references to support the schemes introduced?